engine/docs/src/script_doc.py

```diff
@@ -157,14 +157,26 @@ def _strip_comment_stars(str):
         ret.append(line)
     return '\n'.join(ret)
 
-def _parse_comment(str):
-    str = _strip_comment_stars(str)
+# Converts "<p>text</p>" to "text"
+def _strip_paragraph(text):
+    if text.startswith('<p>'):
+        text = text[3:]
+    if text.endswith('</p>'):
+        text = text[:-4]
+    return text
```

Comment:
The results was very verbose, and I wanted better control inside the documentation generation, as opposed to at this point.